src/libs/antares/study/parts/hydro/container.cpp

```diff
@@ -89,6 +89,8 @@ void PartHydro::reset()
     allocation.clear();
     // allocation.fromArea(<current area>, 1.); // Area::reset()
 
+    overflowCost = 0.;
+
```

Comment:
Should levelCost also have to be reset to 0 in `reset` function?

Reply:
a67c0894d